src/amo/components/SectionLinks/index.js

```diff
@@ -50,16 +56,19 @@ export class SectionLinksBase extends React.Component<Props> {
   };
 
   render() {
-    const { className, clientApp, i18n, viewContext } = this.props;
+    const { _config, className, clientApp, i18n, viewContext } = this.props;
     const isExploring = [VIEW_CONTEXT_EXPLORE, VIEW_CONTEXT_HOME].includes(
       viewContext,
     );
 
     let forBrowserNameText;
+    let showThemes = true;
```

Comment:
maybe `const showThemes = shouldShowThemes({ _config, clientApp }))` ? or maybe you can just wrap shouldShowThemes itself around the li without assigning it since it's only used 1 time here?